packages/account/src/Containers/account.jsx

```diff
@@ -113,7 +115,14 @@ const Account = ({
     routes.forEach(menu_item => {
         menu_item.subroutes.forEach(route => {
             if (route.path === shared_routes.financial_assessment) {
-                route.is_disabled = is_virtual;
+                if (active_account_landing_company === 'maltainvest') {
+                    route.is_disabled = is_virtual || !is_risky_client;
+                } else {
+                    route.is_disabled = is_virtual;
+                }
+            }
+            if (route.path === shared_routes.trading_assessment) {
+                route.is_disabled = is_virtual || active_account_landing_company !== 'maltainvest';
```

Comment:
@likhith-deriv  How should be the behaviour of trading assessment in Account settings for CR clients? Is it  hidden / enabled /disabled ?